.devcontainer/Dockerfile

```
@@ -4,20 +4,19 @@ FROM golang:latest
 
 ENV GOPATH=/go
 ENV PATH=$PATH:$GOPATH/bin
-ENV AVALANCHEGO_EXEC_PATH=$GOPATH/src/github.com/ava-labs/avalanchego/build/avalanchego
 ENV AVALANCHEGO_PLUGIN_PATH=$GOPATH/src/github.com/ava-labs/avalanchego/build/plugins
 
 # Install Node.js and npm using the official Node.js image
 RUN curl -fsSL https://deb.nodesource.com/setup_20.x | bash - && apt-get install -y nodejs
 
-# Clone the avalanchego repository
-RUN git clone -b v1.10.5 https://github.com/ava-labs/avalanchego.git $GOPATH/src/github.com/ava-labs/avalanchego
+WORKDIR /
 
-# Set the working directory to the cloned repository
-WORKDIR $GOPATH/src/github.com/ava-labs/avalanchego
+# # Build the avalanchego project using the sh script
+RUN curl -LJO https://github.com/ava-labs/avalanchego/releases/download/v1.10.5/avalanchego-linux-amd64-v1.10.5.tar.gz
```

would it make sense to define the version of avalanche as an ENV?

```
ENV AVALANCHEGO_VERSION=1.10.5
```
```suggestion
RUN curl -LJO https://github.com/ava-labs/avalanchego/releases/download/v$[AVALANCHEGO_VERSION}/avalanchego-linux-amd64-v$[AVALANCHEGO_VERSION}.tar.gz
```